capabilities/prioritization/README.md

```diff
@@ -8,27 +8,27 @@ High priority requests should always be allowed through to the backend, while lo
 An example use for this capability would be when you have purchased provisioned throughput and want to utilize any spare capacity for low-priority requests.
 
 There are two implementations of the prioritization policy:
-- [Simple](./prioritization-simple.md)
-- [Token-counting](./prioritization-token-counting.md)
+- [Token tracking](./prioritization-token-tracking.md)
+- [Token calculating](./prioritization-token-calculating.md)
 
-## Simple vs token-counting approaches
+## Token-tracking vs token-calculating approaches
 
-The two approaches are largely similar, but differ in the way in which they calculate the number of tokens a request should consume, for rate limiting purposes. The token counting approach calculates the numer of tokens using a policy expression with logic meant to approximate Azure Open AI Service's internal algorithm, while the simple approach relies on the returned headers from the AOAI service to set remaining capacity. The main concerns when selecting an approach to adopt are as follows:
+The two approaches are largely similar, but differ in the way in which they determine the available token capacity for rate limiting purposes. The token calculating approach calculates the numer of tokens using a policy expression with logic meant to approximate Azure Open AI Service's internal algorithm, while the token tracking approach relies on the returned headers from the AOAI service to set remaining capacity. The main concerns when selecting an approach to adopt are as follows:
 
-- Simple approach introduces a lag when rate-limited by the backend (especially when no high-priority requests are processed). Token counting approach returns immediate 429s to callers given tokens are calculated internally, rather than waiting on returned AOAI headers.
-- Token counting approach adds maintenance costs to ensure consumed token calculation remains up to date with AOAI internal logic.
+- Token-tracking approach introduces a lag when rate-limited by the backend (especially when no high-priority requests are processed). The token calculting approach returns immediate 429s to callers given tokens are calculated internally, rather than waiting on returned AOAI headers.
```

Comment:
```suggestion
- Token-tracking approach introduces a lag when rate-limited by the backend (especially when no high-priority requests are processed). The token calculating approach returns immediate 429s to callers given tokens are calculated internally, rather than waiting on returned AOAI headers.
```